textattack/augmentation/recipes.py

```diff
@@ -142,3 +142,41 @@ def __init__(self, **kwargs):
             ]
         )
         super().__init__(transformation, constraints=DEFAULT_CONSTRAINTS, **kwargs)
+
+
+class CheckListAugmenter(Augmenter):
+    """Augments words by using the transformation methods provided by CheckList
+    INV testing, which combines:
+
+    - Name Replacement
+    - Location Replacement
+    - Number Alteration
+    - Contraction/Extension
+
+    "Beyond Accuracy: Behavioral Testing of NLP models with CheckList" (Ribeiro et al., 2020)
+    https://arxiv.org/abs/2005.04118
+    """
+
+    def __init__(self, **kwargs):
+        from textattack.transformations import (
+            CompositeTransformation,
+            WordSwapChangeLocation,
+            WordSwapChangeName,
+            WordSwapChangeNumber,
+            WordSwapContract,
+            WordSwapExtend,
+        )
+
+        transformation = CompositeTransformation(
+            [
+                WordSwapChangeNumber(),
+                WordSwapChangeLocation(),
+                WordSwapChangeName(),
+                WordSwapExtend(),
+                WordSwapContract(),
+            ]
+        )
+
+        constraints = [DEFAULT_CONSTRAINTS[0]]
```

Comment:
Do you think it makes sense to include both pretransformation constraints in this case?

Reply:
I think if we include `StopwordModification` as constraints, `WordSwapContract` will be handicapped because words like "I" or "you" are stop words. So when we try to contact "I will" to "I'll", `WordSwapContract` will be stopped by `StopwordModification`

Reply:
Ah good call!